Sample/SextantSample.Android/SextantSample.Android.csproj

```diff
@@ -59,6 +59,9 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="System.Reactive">
+      <Version>4.0.0</Version>
```

Comment:
Worth changing to 4.1.5+, as that's where we are going from now on.

Reply:
When I moved this to 4.1.5 I got a fun error about a Task nuget.  I moved it back to 4.0.0 so I could test the sample.